tests/test_opera_s1_rtc.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+
+import hyp3_sdk
+import pytest
+import requests
+from osgeo import gdal
+
+from hyp3_testing import util
+from hyp3_testing.helpers import archive_tifs, job_tifs
+from hyp3_testing.opera_compare import compare_rtc_hdf5_files, compare_rtc_s1_products
+
+
+gdal.UseExceptions()
+CMR_URL = 'https://cmr.earthdata.nasa.gov/search/granules.umm_json'
+pytestmark = pytest.mark.golden
```

Comment:
I don't see `pytestmark` being used elsewhere, what's it for?

Reply:
It marks every test in this module with the `golden` tag.